snuba/consumers/consumer.py

```diff
@@ -499,6 +499,13 @@ def process_message(
     snuba_logical_topic: SnubaTopic,
     message: Message[KafkaPayload],
 ) -> Union[None, BytesInsertBatch, ReplacementBatch]:
+    local_metrics = MetricsWrapper(
+        metrics,
+        tags={
+            "consumer_group": consumer_group,
+            "snuba_logical_topic": snuba_logical_topic.name,
```

Comment:
The physical topic names would be more consistent with those on the dashboard, if we have a way to easily resolve those

Reply:
Possibly but it would be more data to be serialized into subprocesses. I was actually thinking of getting rid of `snuba_logical_topic` too